.github/workflows/contract-audit.yml

```
@@ -31,10 +31,50 @@ jobs:
         run: ./scripts/contract-audit.sh
 
+      - name: Build contracts and calculate WASM hashes
+        run: |
+          cd packages/contracts
+          rustup target add wasm32-unknown-unknown
+          # Build only workspace-member contracts (excludes non-member dirs like rent_escrow)
+          # Use cargo metadata to enumerate workspace package names, then filter to those
+          # whose manifest path lives under contracts/.
+          cargo metadata --no-deps --format-version 1 \
+            | python3 -c "
+import sys, json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)